backend/onedrive/onedrive.go

```diff
@@ -1799,8 +1886,8 @@ func (o *Object) ID() string {
 	return o.id
 }
 
-func newOptsCall(normalizedID string, method string, route string) (opts rest.Opts) {
-	id, drive, rootURL := parseNormalizedID(normalizedID)
+func newOptsCall(normalizedID string, method string, route string, Region string) (opts rest.Opts) {
```

Comment:
```suggestion
func newOptsCall(normalizedID string, method string, route string, region string) (opts rest.Opts) {
```

Lower case by convention